### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by vitalyavolyn**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -463,3 +463,36 @@ DefaultContentLanguage = "en"
     weight = 6
     name = "Twitter"
     url = "https://twitter.com/redox_os"
+
+[languages.ru]
+  weight = 0
```
Comment (jackpot51) [inline config.toml]: The language weight should be changed to identify order by total speakers.

Comment (vitalyavolyn) [inline config.toml]: Which weight should i set?

Comment (jackpot51) [inline config.toml]: One after Spanish, I think. Then increase larger language weights by one

**Comment by jackpot51** [comment]
Thanks!

**Pull request merged**